### DISCUSSION TITLE
Parametric Sweep on a Boundary Condition

### DISCUSSION THREAD

**Discussion by jsevic** [idaholab/moose#28525] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Is there a means of implementing a parametric sweep in MOOSE, specifically parametrizing a boundary condition and then running a Newton transient simulation to steady-state.

What I would like to do is step voltage, a boundary condition, from say 0 V to 1.0 V in steps of 0.1 V and at each voltage step run the executioner with a Netwon transient simulation. Each voltage step would use the previous voltage step as the initial condition.

I can do this manually, it would be nice to automate it.

Thank you.
John Sevic

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can do that with the tools in stochastic tools, or in the optimization module
https://mooseframework.inl.gov/syntax/ParameterStudy/index.html

**Reply by jsevic** [reply]
Great this is exactly what we were looking for thx!
